Animations/Animations.js

```diff
@@ -0,0 +1,194 @@
+// 'use strict';
+var React = require("react/addons");
+var Arrival = require('Arrival');
+var ReactTransitionGroup = React.addons.TransitionGroup;
+
+//makeTransition FUNCTION: creates animation for wrapped component based on props of wrapper component
+  //@param displayName STRING: intended name of the wrapper animation component
+  //@param transitions OBJECT: properties that define the default animation properties for the animation component wrapper
+  //Required keys:
+    // render FUNCTION: since this is an implementation of React.createClass a render method is required
+    // componentWillEnter FUNCTION: required to for animation entering
+    // componentWillLeave FUNCTION: required to for animation leaving
+  //Additional keys
+    // componentDidEnter FUNCTION: use to define actions to run after an animation has entry completed
+    // componentDidEnter FUNCTION: use to define actions to run after an animation has leave completed
+var makeTransition = function(displayName, transitions) {
+
+  return React.createClass({
+    componentWillEnter: function(done) {
+      // This is called at the same time as componentDidMount() for components added to an existing TransitionGroup. It will block other animations from occurring until callback is called. It will not be called on the initial render of a TransitionGroup.
+      this.el = this.getDOMNode();
```

Comment:
Recommend refactoring this line and the next into their own method as it is used in a couple of places.